includes/class-mailer.php

```diff
@@ -135,15 +135,26 @@ public static function new_follower( $notification ) {
 		}
 
 		/* translators: 1: Blog name, 2: Follower name */
-		$subject = \sprintf( \esc_html__( '[%1$s] Follower: %2$s', 'activitypub' ), \esc_html( get_option( 'blogname' ) ), \esc_html( $actor['name'] ) );
-		/* translators: 1: Blog name, 2: Follower name */
-		$message = \sprintf( \esc_html__( 'New Follower: %2$s.', 'activitypub' ), \esc_html( get_option( 'blogname' ) ), \esc_html( $actor['name'] ) ) . "\r\n\r\n";
-		/* translators: Follower URL */
-		$message .= \sprintf( \esc_html__( 'URL: %s', 'activitypub' ), \esc_url( $actor['url'] ) ) . "\r\n\r\n";
-		$message .= \esc_html__( 'You can see all followers here:', 'activitypub' ) . "\r\n";
-		$message .= \esc_url( \admin_url( $admin_url ) ) . "\r\n\r\n";
-
-		\wp_mail( $email, $subject, $message );
+		$subject = \sprintf( \__( '[%1$s] Follower: %2$s', 'activitypub' ), get_option( 'blogname' ), $actor['name'] );
+
+		\ob_start();
+		require ACTIVITYPUB_PLUGIN_DIR . '/templates/new-follower-email.php';
+		$html_message = \ob_get_clean();
+
+		$alt_function = function ( $mailer ) use ( $actor, $admin_url ) {
+			/* translators: 1: Blog name, 2: Follower name */
+			$message = \sprintf( \__( 'New Follower: %2$s.', 'activitypub' ), \get_option( 'blogname' ), $actor['name'] ) . "\r\n\r\n";
```

Comment:
If we don't expect to use the blog name, let's just limit it to one placeholder?